3.-configuring-your-deployment/3.4-data-sources.md

```diff
@@ -58,7 +58,56 @@ To get started with email set up,
 
 Nb: Reports coming via email will take a while before they reflect on the platform because there is a “job” that needs to be run in the database and those jobs  run every 10 minutes or so.
 
-## 3.4.2 FrontlineSMS <a id="3-4-2-frontlinesms-pending"></a>
+## 3.4.2 GMail <a id="3-4-2-gmail"></a>
```

Comment:
@webong  have you checked that these anchor links (i.e. "3-4-2-frontlinesms-pending" are not used anywhere else on the doc?)

Reply:
Just did a search through the docs codebase, it's used nowhere ... also did the check for other anchor tags id.